lib/redcord/vacuum_helper.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# typed: strict
+
+module Redcord::VacuumHelper
+  extend T::Sig
+  extend T::Helpers
+
+  sig { params(model: T.class_of(Redcord::Base)).void }
+  def self.vacuum(model)
```

Comment:
since this is a rake task, might be worth to add some logging  (prints to STDOUT might be good enough for now